images/linux/toolsets/toolset-2004.json

```diff
@@ -259,5 +259,51 @@
     },
     "rubygems": [
         {"name": "fastlane"}
+    ],
+    "node_modules": [
+	    {
+		    "name": "grunt",
+		    "command": "grunt"
+	    },
+	    {
+		    "name": "gulp",
+		    "command": "gulp"
+	    },
+	    {
+		    "name": "n",
+		    "command": "n"
+	    },
+	    {
+	            "name": "parcel-bundler",
+		    "command": "parcel"
+	    },
+	    {
+		    "name": "typescript",
+		    "command": "tsc"
+	    },
+	    {
+		    "name": "newman",
+		    "command": "newman"
+	    },
+	    {
+	            "name": "vercel",
+		    "command": "vercel"
+	    },
+	    {
+	            "name": "webpack",
+		    "command": "webpack"
+	    },
+	    {
+	            "name": "webpack-cli",
+		    "command": "webpack-cli"
+	    },
+	    {
+	            "name": "netlify-cli",
+		    "command": "netlify"
+	    },
+	    {
+	            "name": "lerna",
+		    "command": "lerna"
+	    }
```

Comment:
Could you please revisit all toolset files and replace tabs to spaces?
Looks like you have mixed them in toolset files and in nodejs.sh